spec/concepts/todo/create_spec.rb

```diff
@@ -1,7 +1,11 @@
 require 'rails_helper'
 
 RSpec.describe Todo::Create do
-  it "can be spec'd" do
-    raise "Implement me"
+  it "Should be persisted" do
+    params = {todo: {title: 'Test Title'}}
+    todo = Todo::Create.(params).model
+
+    expect(todo.persisted?).to be_truthy
```

Comment:
Please use true/false when what you're asserting is boolean in nature. Nevertheless, this is correct. Just a little bit not quite. Good job.